R/FilterState.R

```diff
@@ -1,16 +1,16 @@
-#' @name FilterState
+#' @name InteractiveFilterState
```

Comment:
Should we change the file name to `InteractiveFilterState.R` for consistency on the file name of abstract class?